packages/azure-openapi-validator/autorest/src/spectral-plugin-func.ts

```diff
@@ -36,7 +36,8 @@ export async function spectralPluginFunc(initiator: IAutoRestPluginInitiator): P
   }
 
   const resolvedOpenapiType: OpenApiTypes = getOpenapiType(openapiType)
-  const [ruleset, namesOfRulesInStagingOnly]: [Ruleset, string[]] = await getRuleSet(resolvedOpenapiType)
+  // FIXME: This should be set by the CI, not hardcoded
```

Comment:
Can you elaborate what is the `this` here?

Reply:
The second parameter is hardcoded to `true`, it should be set to `does swagger contain x-typespec-generated`.

Reply:
Yeah, I think it would be cool if the comment in the code spells that out.